packages/repeater/src/lib/RepeaterFactory.spec.ts

```diff
@@ -115,5 +121,51 @@ describe('RepeaterFactory', () => {
       expect(arg?.description).toBe('description');
       expect(res).toBeInstanceOf(Repeater);
     });
+
+    it('should create repeater and apply RequestRunnerOptions', async () => {
+      const factory = new RepeaterFactory(configuration);
+      when(
+        mockedContainer.register(RequestRunnerOptions, anything())
+      ).thenReturn();
+
+      const requestRunnerOptions = {
+        timeout: 10000,
+        maxContentLength: 200,
+        reuseConnection: false,
+        whitelistMimes: [
+          'text/html',
+          'text/plain',
+          'text/css',
+          'text/javascript',
+          'text/markdown',
+          'text/xml',
+          'application/javascript',
+          'application/x-javascript',
+          'application/json',
+          'application/xml',
+          'application/x-www-form-urlencoded',
+          'application/msgpack',
+          'application/ld+json',
+          'application/graphql'
+        ]
+      };
+
+      const res = await factory.createRepeater({
+        namePrefix: 'foo',
+        description: 'description',
+        requestRunnerOptions
+      });
+
+      verify(
+        mockedContainer.register(
+          RequestRunnerOptions,
+          objectContaining({
+            useValue: requestRunnerOptions
+          })
+        )
+      ).once();
```

Comment:
this check is not enough. a "container" registered in created child container - a kind of newly introduced implicit public api, which is mandatory for other parts (bus & handlers) to work as expected -  and thus should be explicitly mentioned/checked here in unit tests